Makefile

```diff
@@ -134,6 +134,7 @@ generate: $(CONTROLLER_GEN) ## Generate code
 generate-go: $(CONTROLLER_GEN) $(CONVERSION_GEN) ## Runs Go related generate targets
 	$(CONTROLLER_GEN) \
 		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+		crd:preserveUnknownFields=false \
```

Comment:
We shouldn't have this line here - this invocation of controller-gen is to generate the DeepCopy go functions. We generate CRDs in the `generate-manifests` target.